docs/odcs.rst

```diff
@@ -24,84 +24,188 @@ idea is to not require the user to provide a yum repo url when building
 container images.  Instead, OSBS will use ODCS to generate a yum repo with the
 relevant signed packages based on user configuration.
 
+Concepts
+--------
+
+Signing Intent
+""""""""""""""
+
+When an ODCS compose is used, the idea of a signing intent is used. This refers
+to the signature of the content to be included in the container image. For
+instance, an environment may provide the following intents: ``release``,
+``beta``, and ``unsigned``. Each one of those intents is then mapped to a list
+of signing keys. These signing keys are then used during compose creation. The
+packages to be included must have been signed by either one of the keys listed.
```

Comment:
"either one" reads a little awkwardly there. Maybe "any"?

Reply:
Changed it to "any".